<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HP 8751A Suite</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnLoopgain">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Loop gain and phase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImpedance">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Impedance analysis</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnRetry">
         <property name="text">
          <string>Retry</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSettings">
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="status">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
